api-reference/10 UI Widgets/dxForm/5 Item Types/SimpleItem/template.md

```diff
@@ -32,6 +32,203 @@ A template name or container.
 
 ---
 
+In Angular, Vue, and React, this template can be used to configure a custom editor more natively than using [editorType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorType) and [editorOptions](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorOptions). When you configure a custom editor in the template, consider the following specificities:
```

Comment:
"more natively" Clarify

Reply:
```suggestion
In Angular, Vue, and React, this template can be used instead of [editorType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorType) and [editorOptions](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorOptions)  to configure a custom editor. It gives you the advantage of using nested configuration components. When you configure a custom editor in the template, consider the following specificities:
```

Reply:
Ok